qiskit_nature/drivers/molecule.py

```diff
@@ -70,6 +70,19 @@ def __init__(
 
         self._perturbations = None  # type: Optional[List[float]]
 
+    def __str__(self) -> str:
```

Comment:
It might have been convenient for this to simply have been the standard xyz format for a molecule definition - though that would have meant not having masses as I don't think that's part of it - or at least I guess it could be appended afterwards. One reason for mentioning using that is then the extra labelling for geometry, multiplicity etc would not be needed, since its standard, and this whole output would be more compact.

Reply:
According to [Wikipedia](https://en.wikipedia.org/wiki/XYZ_file_format) the XYZ format has neither support for masses nor spins or charges. It links to an extended format but that requires information like cell vectors (which we do not have or which does not always apply).

Thus, I am not 100% sure what you are saying: I agree, that the xyz format could have been useful but we cannot, in any way, add all the information which we have here. Besides: the `geometry` section is formatted as XYZ with only the top line indicating the number of atoms missing. But that can be easily added by a user.

Is this okay as is for now or what concretely would you like me to change?

Reply:
I guess I was more thinking of the xyz in the context drivers do it e.g
```
0 1
H 0 0 0
H 0 0 0.75
```
and yes I was saying that does not include mass. But having seen the Property notebook and the formatting there I guess the 'extra' verbosity in the str here fits in general with what you have here. So lets go with this then as you have it.